test/scripts/setup-contour-provisioner.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+# set -o pipefail
+set -o errexit
+set -o nounset
+
+readonly KIND=${KIND:-kind}
+readonly KUBECTL=${KUBECTL:-kubectl}
+
+readonly CLUSTERNAME=${CLUSTERNAME:-contour-e2e}
+readonly WAITTIME=${WAITTIME:-5m}
+
+readonly HERE=$(cd $(dirname $0) && pwd)
+readonly REPO=$(cd ${HERE}/../.. && pwd)
+
+kind::cluster::exists() {
+    ${KIND} get clusters | grep -q "$1"
+}
+
+kind::cluster::load::docker() {
+    ${KIND} load docker-image \
+        --name "${CLUSTERNAME}" \
+        "$@"
+}
+
+if ! kind::cluster::exists "$CLUSTERNAME" ; then
+    echo "cluster $CLUSTERNAME does not exist"
+    exit 2
+fi
+
+# Create the Contour namespace and service account.
+${KUBECTL} apply -f ${REPO}/examples/contour/namespace.yaml
+${KUBECTL} apply -f ${REPO}/examples/contour/service-account.yaml
+
+# Set the image tag to match the current Contour version.
+VERSION="v$$"
+
+# Build the Contour Provisioner image.
+make -C ${REPO}/provisioner container IMAGE=ghcr.io/projectcontour/contour-provisioner VERSION=${VERSION}
```

Comment:
The image name used here should still be `ghcr.io/projectcontour/contour` (on this line and other places) and `-C ${REPO}/provisioner` should be `-C ${REPO}`